lib/ansible/module_utils/eos.py

```diff
@@ -139,8 +143,9 @@ def get_config(self, flags=[]):
         except KeyError:
             conn = get_connection(self)
             rc, out, err = self.exec_command(cmd)
+            out = to_text(out, errors='surrogate_or_strict')
             if rc != 0:
-                self._module.fail_json(msg=err)
+                self._module.fail_json(msg=to_text(err, errors='surrogate_or_strict'))
```

Comment:
Note: The msg to fail_json and exit_json is okay to use the default error handler (surrogate_then_replace).  The reason is that it's only going to be used to display information to the user.  We need to use a strict variant when we'd rather traceback then to change the string for unknown bytes (examples, if the data is part of a filename, it's used as a database key, or we don't know what's going to happen to the data later.)